### ISSUE TITLE
Errors when running New-UDDesktopApp

### ISSUE THREAD

**Issue by camusicjunkie** [ironmansoftware/ud-forge#1]
I just want to say first that I'm really not familiar with nodejs or electron.

Are there any prerequisites on folder structure or maybe nodejs version? When I follow the usage example on the readme I just get a bunch of errors. One saying the package.json doesn't exist and another that the index.js doesn't exist. Below is the log file from npm after running this function. Based on that log file it seems it's trying to find those files in a broken path of my user directory.

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   'create-electron-app@latest',
1 verbose cli   '--global',
1 verbose cli   '--prefix',
1 verbose cli   'C:\\Users\\John',
1 verbose cli   'Steele\\AppData\\Roaming\\npm-cache\\_npx\\15820',
1 verbose cli   '--loglevel',
1 verbose cli   'error',
1 verbose cli   '--json' ]
2 info using npm@6.9.0
3 info using node@v10.16.3
4 verbose npm-session 77bd11efd395e579
5 silly install loadCurrentTree
6 silly install readGlobalPackageData
7 silly fetchPackageMetaData error for file:Steele\AppData\Roaming\npm-cache\_npx\15820 Could not install from "Steele\AppData\Roaming\npm-cache\_npx\15820" as it does not contain a package.json file.
8 http fetch GET 304 https://registry.npmjs.org/create-electron-app 174ms (from cache)
9 silly pacote tag manifest for create-electron-app@latest fetched in 188ms
10 timing stage:rollbackFailedOptional Completed in 1ms
11 timing stage:runTopLevelLifecycles Completed in 196ms
12 verbose stack Error: ENOENT: no such file or directory, open 'E:\PowerShell\Modules\Out\Steele\AppData\Roaming\npm-cache\_npx\15820\package.json'
13 verbose cwd E:\PowerShell\Modules\Out
14 verbose Windows_NT 10.0.18362
15 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "create-electron-app@latest" "--global" "--prefix" "C:\\Users\\John" "Steele\\AppData\\Roaming\\npm-cache\\_npx\\15820" "--loglevel" "error" "--json"
16 verbose node v10.16.3
17 verbose npm  v6.9.0
18 error code ENOLOCAL
19 error Could not install from "Steele\AppData\Roaming\npm-cache\_npx\15820" as it does not contain a package.json file.
20 verbose exit [ 1, true ]
```

**Comment by adamdriscoll** [comment]
I'm running these node\npm versions. Seems like you should be fine. 
```
PS C:\Users\adamr> npx --version
6.10.3
PS C:\Users\adamr> node --version
v10.16.3
```

I just released a new version after finds some issues. Can you upgrade and try again? I also added Write-Verbose in there so switch that on and we'll see if we can figure this out.

**Comment by camusicjunkie** [comment]
The npx version I had was 6.9. I updated npm but that still didn't work. I found a workaround but it definitely seems like npx is broken on my machine.

I think the problem is occurring because when npx runs npm install internally the --prefix flag is using a broken path. My user path has a space it isn't accounting for.
```
"--prefix" "C:\\Users\\John" "Steele\\AppData\\Roaming\\npm-cache\\_npx\\15820"
```

What I did to get around that was to change the module as seen below:
```
npm install create-electron-app@latest --global
npx create-electron-app $Name
```
Once I reran ```New-UDDesktopApp -Path .\dashboard.ps1 -Name Test -OutputPath Out -Verbose``` with the needed changes to the dashboard it runs just fine.

![image](https://user-images.githubusercontent.com/33888807/65710366-bbd35580-e047-11e9-9199-c6a310b08d00.png)
